src/services/SessionService.ts

```diff
@@ -0,0 +1,49 @@
+import { injectable, inject } from 'inversify';
+import { provide } from 'inversify-binding-decorators';
+import { Session } from '../models/Session';
+import * as crypto from 'crypto';
+import { IMap } from 'src/models/types';
+import { RedisService } from './RedisService';
+import { PenaltyReferenceDetails } from '../models/PenaltyReferenceDetails';
+
+
+@provide(SessionService)
+export class SessionService {
```

Comment:
This will be thrown away so I suggest you do not spend much more time polishing it.